pkg/transform/transform.go

```diff
@@ -1,8 +1,10 @@
 package transform
 
 import (
+	"fmt"
 	"github.com/expect-digital/translate/pkg/model"
 	pb "github.com/expect-digital/translate/pkg/server/translate/v1"
+	"golang.org/x/text/language"
 )
 
 func MessageFromProto(m *pb.Message) model.Message {
```

Comment:
It's better to handle all errors in go lang. Let's return an error if the language code (BCP 47) is invalid.

```suggestion
func MessageFromProto(m *pb.Message) (model.Message, error) {
```